src/ncp/spinel.h

```diff
@@ -2003,6 +2003,23 @@ typedef enum {
      */
     SPINEL_PROP_CHILD_SUPERVISION_CHECK_TIMEOUT = SPINEL_PROP_OPENTHREAD__BEGIN + 11,
 
+    /// Thread parent response info
+    /** Format: `ESccCCCb` - Read only
+     *
+     *  `E`: Extended address
+     *  `S`: RLOC16
+     *  `c`: Instant RSSI
+     *  'c': Parent Priority
+     *  `C`: Link Quality3
+     *  `C`: Link Quality2
+     *  `C`: Link Quality1
+     *  'b': Is the node receiving parent response frame attached
+     *
+     * This property is available only for FTD builds.
+     *
+     */
+    SPINEL_PROP_PARENT_RESPONSE_INFO = SPINEL_PROP_OPENTHREAD__BEGIN + 13,
+
```

Comment:
Can you please update `spinel.c` with new property in `spinel_prop_key_to_cstr()`?

Reply:
Added